tests/notificationprofile/test_models.py

```diff
@@ -163,6 +163,43 @@ def test_incident_fits_maxlevel(self):
         self.assertEqual(result, level <= maxlevel)
 
 
+@tag("unittest")
+class FilterWrapperEventTypeEmptyTests(unittest.TestCase):
+    # Validation is handled before the data gets to FilterWrapper
+    # An event type must be one of the types in Event.Type if it is set at all.
+
+    def test_is_event_type_empty_is_false(self):
+        empty_filter = FilterWrapper({})
+        result = empty_filter.is_event_type_empty()
+        self.assertTrue(result)
+
+    def test_is_event_type_empty_is_true(self):
+        empty_filter = FilterWrapper({"event_type": "whatever"})
+        result = empty_filter.is_event_type_empty()
+        self.assertFalse(result)
```

Comment:
Ditto as @hmpf suggests.

Also, IMHO, I find these tests would read more nicely without the intermediate `result` variable:

```suggestion
    def test_when_filter_is_empty_is_event_type_empty_should_return_true(self):
        empty_filter = FilterWrapper({})
        self.assertTrue(empty_filter.is_event_type_empty())

    def test_when_event_filter_exists_is_event_type_empty_should_return_false(self):
        empty_filter = FilterWrapper({"event_type": "whatever"})
        self.assertFalse(empty_filter.is_event_type_empty())
```